src/containers/Debug/fusionChartDemo.js

```diff
@@ -0,0 +1,256 @@
+        "plotspacepercent": "60",
+        "plottooltext": "<b><u>Time taken to complete \"$label\"</u>:</b><br> <br> Max: $maxDataValue mins <br> Min: $minDataValue mins <br> <br> Mean: $mean mins <br> <br> Q3: $Q3 mins <br> Median: $median mins <br> Q1: $Q1 mins"
+    },
+    "categories": [
+        {
+        "category": [
+            {
+            "label": "1. Introduction"
+            },
+            {
+            "label": "2. Setup"
+            },
+            {
+            "label": "3. Props"
+            },
+            {
+            "label": "4. Components"
+            },
+            {
+            "label": "5. Redux"
+            },
+            {
+            "label": "6. DOM Events"
+            },
+            {
+            "label": "7. Conditional Output"
+            },
+            {
+            "label": "8. CSS Files"
+            }
+        ]
+        }
+    ],
+    "dataset": [
+        {
+        "seriesname": "Interquartile Range",
+        "data": [
+            {
+            "value": calcOutlier([5, 10, 8, 6, 7, 9, 13, 5, 17, 100],"value"),   // Both uses the same list of data
+            "outliers": calcOutlier([5, 10, 8, 6, 7, 9, 13, 5, 17, 100],"outlier")
+            },
+            {
+            "value": calcOutlier([2, 4, 2, 5, 3, 6, 4, 5, 30],"value"),
+            "outliers": calcOutlier([2, 4, 2, 5, 3, 6, 4, 5, 3],"outlier")
+            },
+            {
+            "value": calcOutlier([21, 20, 26, 27, 30, 40, 50, 35, 10, 70],"value"),
+            "outliers": calcOutlier([21, 20, 26, 27, 30, 40, 50, 35, 10, 70],"outlier")
+            },
+            {
+            "value": calcOutlier([23, 25, 21, 32, 36, 29, 31, 29, 29, 46], "value"),
+            "outliers": calcOutlier([23, 25, 21, 32, 36, 29, 31, 29, 29, 46], "outlier")
+            },
+            {
+            "value": calcOutlier([60, 70, 67, 65, 80, 66, 62, 22, 120, 100],"value"),
+            "outliers": calcOutlier([60, 70, 67, 65, 80, 66, 62, 22, 120, 100],"outlier")
+            },
+            {
+            "value": calcOutlier([33, 37, 39, 29, 52, 47, 32, 35, 5, 90],"value"),
+            "outliers": calcOutlier([33, 37, 39, 29, 52, 47, 32, 35, 5, 90],"outlier")
+            },
+            {
+            "value": calcOutlier([55, 57, 59, 59, 42, 47, 42, 45, 60, 59],"value"),
+            "outliers": calcOutlier([55, 57, 59, 59, 42, 47, 42, 45, 60, 59],"outlier")
+            },
+            {
+            "value": calcOutlier([70, 77, 79, 89, 72, 87, 72, 75, 20, 120],"value"),
+            "outliers": calcOutlier([70, 77, 79, 89, 72, 87, 72, 75, 20, 120],"outlier")
+            }
+        ]
+        }
+    ]
+};
+
+const chartConfigs = {  // chartConfig is for the first main chart you see
+    type: 'boxandwhisker2d',
+    width: '100%',
+    height: 500,
+    dataFormat: 'json',
+    dataSource: myDataSource,
+};  
+
+
+const mapStateToProps = (state, ownProps) => ({
+    uid: state.firebase.auth.uid,
```

Comment:
do you need these states from the redux store in this component?

Reply:
I don't think so. I will be making the changes